src/plus/webviews/graph/graphWebview.ts

```diff
@@ -113,76 +116,78 @@ export class GraphWebview extends WebviewWithConfigBase<State> {
 			return undefined;
 		}
 
-		const [currentUser, log, /*remotes,*/ tags, branches] = await Promise.all([
-			this.container.git.getCurrentUser(repo as string),
-			this.getLog(this.selectedRepository as string),
-			// this.container.git.getRemotes(repo as string),
-			this.container.git.getTags(repo as string),
-			this.container.git.getBranches(repo as string),
-		]);
-
-        if (log == null || log?.commits === undefined) {
-			return undefined;
-        }
-
-        this.currentLog = log;
-
-		const name = currentUser?.name ? `${currentUser.name} (you)` : 'You';
-
-		const commitList: any[] = [];
-
-		for (const commit of log.commits.values()) {
-			const commitBranch = branches.values.find(b => b.sha === commit.sha);
-			let branchInfo = {} as any;
-			if (commitBranch != null) {
-				branchInfo = {
-					remotes: [
-					{
-						name: commitBranch.name,
-						url: commitBranch.id
-					}
-					]
-				};
-				if (commitBranch.current) {
-					branchInfo.heads = [
-						{
-							name: commitBranch.name,
-							isCurrentHead: true
-						}
-					];
-				}
+		if (this.currentLog === undefined) {
+			const log = await this.getLog(this.selectedRepository);
+			if (log?.commits === undefined) {
+				return undefined;
 			}
-			const commitTag = tags.values.find(t => t.sha === commit.sha);
-			let tagInfo = {} as any;
-			if (commitTag != null) {
-				tagInfo = { tags: [
-						{
-							name: commitTag.name,
-						}
-					]
-				};
-			}
-			commitList.push({
-				sha: commit.sha,
-				parents: commit.parents,
-				author: commit.author.name === 'You' ? name : commit.author.name,
```

Comment:
@miggy-e FYI: I think this logic is not needed

Reply:
i think this was part of copied code from some other place, I'm fine with removing the logic